idea/idea-live-templates/src/org/jetbrains/kotlin/idea/liveTemplates/KotlinTemplateContextType.java

```diff
@@ -162,11 +163,15 @@ public Statement() {
 
         @Override
         protected boolean isInContext(@NotNull PsiElement element) {
-            PsiElement parentStatement = PsiTreeUtil.findFirstParent(element, new Condition<PsiElement>() {
-                @Override
-                public boolean value(PsiElement element) {
-                    return element instanceof KtExpression && (element.getParent() instanceof KtBlockExpression);
-                }
+            PsiElement parentStatement = PsiTreeUtil.findFirstParent(element, e -> {
+                if (!(e instanceof KtExpression)) return false;
+                PsiElement parent = e.getParent();
+                ASTNode node = parent.getNode();
+                return parent instanceof KtBlockExpression ||
+                       parent instanceof KtContainerNodeForControlStructureBody ||
```

Comment:
I've found `KtPsiUtil.isStatementContainer` as on of the usages for `KtContainerNodeForControlStructureBody`, can it be used here? 
Are checks for `THEN` and `ELSE` needed? It looks like `parent instanceof KtContainerNodeForControlStructureBody` should work for them too.

Reply:
Fixed